stable/jenkins/README.md

```diff
@@ -374,6 +377,32 @@ rbac:
   create: true
 ```
 
+### Config as Code with additionl agents
```

Comment:
I've been testing the updated PR. I think somewhere in the README.md you should state that for this feature to work the following values must set.
```
master.JCasC.enabled: true
master.JCasC.defaultConfig: true
master.JCasC.configScripts: at least one key (cannot be empty {})
```
Specifically the last one. I still don't know why, but if that value is empty the additional agents are not created. It also happened in my PR.

Reply:
`master.JCasC.configScripts: at least one key (cannot be empty {})` is not needed anymore

Reply:
Could please elaborate? I don't get to understand.

Reply:
I meant that it's no longer necessary to specify configScripts when enabling JCasC.
https://github.com/helm/charts/issues/21451#issuecomment-601859408

https://github.com/helm/charts/pull/21556 solved that issue.

Reply:
Thanks for the clarification. I've tried this PR and if I don't set a key for configScripts I don't get the additional agents.
Have you tested this PR with `master.JCasC.configScripts: {}` and got the additional agents?

If affirmative, would you mind sharing your values.yaml files used for the test?

Reply:
yes